<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);font:bold 16px</string>
    </property>
    <property name="text">
     <string>Enter a seed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);font:bold 16px</string>
    </property>
    <property name="text">
     <string>Enter a goal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartpushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;color:black;font: bold 16px</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SeedspinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RestartpushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitpushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="WinnerLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LoserLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>381</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="TargetspinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>45</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>65536</number>
    </property>
   </widget>
   <widget class="QLabel" name="Targetlabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
